src/stores/AccountController.ts

```diff
@@ -66,9 +68,14 @@ export const handleVisibilityChange = () => {
 
 export const initializeAccount = async () => {
   await useService(async ({ accountService, config }) => {
+    if (!accountService) {
+      useAccountStore.setState({ loading: false });
+      return;
+    }
     useAccountStore.setState({
       loading: true,
       canUpdateEmail: accountService.canUpdateEmail,
+      canSupportEmptyFullName: accountService.canSupportEmptyFullName,
```

Comment:
Do these properties all need to go in the store if they are always still available on the service anyway?